app/src/tests/components/contents/packets/PacketReports.test.tsx

```diff
@@ -2,14 +2,19 @@ import { render, waitFor, screen } from "@testing-library/react";
 import { PacketReports } from "../../../../app/components/contents/packets/PacketReports";
 import { mockPacket } from "../../../mocks";
 import { PacketMetadata } from "../../../../types";
+import { Accordion } from "../../../../app/components/Base/Accordion";
 
 jest.mock("../../../../lib/download", () => ({
   getFileObjectUrl: async () => "fakeObjectUrl"
 }));
 
 describe("Packet reports component", () => {
   const renderComponent = (packet: PacketMetadata) => {
-    render(<PacketReports packet={packet} />);
+    render(
+      <Accordion type="single" defaultValue="reports">
+        <PacketReports packet={packet} />
+      </Accordion>
+    );
```

Comment:
I guess a similar change should be made to the PacketParameters tests?

Reply:
This change is still missing

Reply:
shoot missed thanks done